source/versions/filenames.rst

```diff
@@ -0,0 +1,121 @@
+=============================
+Reading the Download Filename
+=============================
+
+When you download files, the name of the file will provide some important version information. The following sections 
+describe the information provided.
+
+SpongeAPI
+=========
+
+The format of the filename is:
+
+``spongeapi-<APIMajor>.<LatestAPIMinorRelease>-shaded.jar``
+
+**OR**
+
+``spongeapi-<APIMajor>.<LatestAPIMinorRelease>.<Date>.<Time>-<BuildNumber>-shaded.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``APIMajor``         | The major version of SpongeAPI implemented by this file (the ``X`` in                         |
+|                      | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``LatestAPI``        | The minor version of SpongeAPI implemented by this file (the ``Y`` in                         |
+| ``MinorRelease``     | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Date``             | The date when the build job ran.                                                              |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Time``             | The time when the build job ran.                                                              |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``<BuildNumber>``    | The build number of Sponge. Supply this number when reporting bugs or seeking support.        |
++----------------------+-----------------------------------------------------------------------------------------------+
+
+.. note::
+
+    The first format without the build information is the release and recommended build format. The second format with 
+    build information is the *SNAPSHOT* format. This version is still in development.
+    
+    See our 
+    `build system 
+    <https://docs.spongepowered.org/stable/en/plugin/buildsystem.html#creating-a-plugin-without-a-build-system>`_ page 
+    for an explanation of the ``-shaded`` label.
+
+.. _sponge-forge-file-name:
+
+SpongeForge
+===========
+
+.. note::
+
+    Sponge changed our versioning policy in October 2018. **Beginning with** SpongeAPI 8.0, the following information 
+    applies. See :doc:`legacy-versions` for information on older versions.
+
+When you download SpongeForge, the name of the file includes a Forge build number which this version of SpongeForge is 
```

Comment:
I know this is old, but this isn't really true, would be nice if you can update this. The version in the filename is the version SpongeForge was built with, that guarantees compatibility, however most of the time newer Forge version also work fine but are not officially supported. So basically just word it differently, same applies to the same section on the old versioning page.

Reply:
Done